build/deploy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM busybox
+FROM buildpack-deps:jessie-curl
```

Comment:
We need *a* `libc.so`; I'm not sure whether we specifically require glibc or if a dynamically-linked musl would work. There are also "alpine with glibc" images out there which might work, but seem kind of hacky.